source/includes/inkling-reference/_curriculum.html.md

```diff
@@ -83,49 +88,62 @@ end
+  Int8{0, 1, 2} action # these values describe game moves
 end
 
 concept high_score
   is classifier
   predicts (Action)
-  follows input(GameState)    feeds outputend
+  follows input(GameState)    
+  feeds output
+end
 ```
 
-The concept high_score trains the Brain to select the next move, which is an Action. Note that the predefined input stream **input** has the schema GameState. This reflects that fact that the simulator has state. The previous move is the state which is input into the selection of the next move.
+The concept `high_score` trains the Brain to select the next move, which will have
+one of the values specified in the `Action` schema range expression. 
 
-The **predicts** schema Action also appears in the simulator clause discussed above. It is after the keyword **control**. In general the **control** schema is the **predicts** schema of the concept being trained.
+Note that the predefined input stream **input** has the schema `GameState`. This reflects that fact that the simulator has state. The previous move is the state which is input into the selection of the next move.
 
-So far we have presented a simple version of the curriculum. Inkling supports multiple simulators and generators within a single curriculum. Here is the full syntax for the curriculum statement, which introduces a usingClause and a withClause (where **using** and **with** will specify simulators). These were not needed in our example above because we were using a single simulator.
+The **predicts** schema `Action` also appears in the simulator clause discussed above. It is after the keyword **control**. In general the **control** schema is the **predicts** schema of the concept being trained.
+
+So far we have presented a simple version of the curriculum. Inkling supports
+multiple simulators and generators within a single curriculum. Here is the full
+syntax for the curriculum statement, which introduces a **using** clause and a
+**with** clause (where **using** and **with** will specify simulators). These were not needed in our example above because we were using a single simulator.
 
 ## Curriculum Rules
 
-* One curriculum per concept. Also, every concept must have a curriculum.
-* You can train with **data**, **simulators**, or **generators**. These are the values allowed as training specifiers. Also, every simulator or generator must be declared with a simulator or generator clause, respectively.
+* One curriculum per concept. 
+* Every concept must have a curriculum.
+* You can train with **data**, **simulators**, or **generators**. These are the values allowed as training specifiers (see the Curriculum syntax). 
+* Every simulator or generator must be declared with a simulator or generator clause, respectively.
 
 **Note:** Currently, during our private beta, you can **only** train with simulators as your training material. That is, only the **simulator** training specifier is supported.
```

Comment:
I think this is the only one I want to insist on changing as we don't call it training material anywhere else.